scripts/vale/vale.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const upath = require( 'upath' );
+const minimist = require( 'minimist' );
+const { globSync } = require( 'glob' );
+const { spawn } = require( 'child_process' );
+
+const ARGS_CHAR_LIMIT = 4000;
```

Comment:
I think we can safely increase the limit to 8000.

- MacOS limit is 262144.
   - https://serverfault.com/questions/69430/what-is-the-maximum-length-of-a-command-line-in-mac-os-x
- Linux limit is 131072.
   - https://unix.stackexchange.com/a/120652
- Windows limit is 8191.
   - https://support.microsoft.com/en-us/help/830473/command-prompt-cmd-exe-command-line-string-limitation